### PR TITLE
AJ-1372: Prune minnie-kenny.sh and git-secrets

### PR DESCRIPTION
**Pull request by jladieu**
Ticket: [AJ-1372](https://broadworkbench.atlassian.net/browse/AJ-1372)

MinnieKenny existed to automate the process of setting up a Rawls workspace to automatically check commits for leaking secrets.  

Per [#dsp-infosec-champions](https://broadinstitute.slack.com/archives/CADU7L0SZ/p1696347522757259) git-secrets and the infrastructure around it are no longer the supported / maintained by AppSec.

---

**PR checklist**

- [x] Include the JIRA issue number in the PR description and title
- [x] ~Make sure Swagger is updated if API changes~
  - [x] ~**...and Orchestration's Swagger too!**~
- [x] ~If you changed anything in `model/`, then you should [publish a new official `rawls-model`](https://github.com/broadinstitute/rawls/blob/develop/README.md#publish-rawls-model) and update `rawls-model` in [Orchestration's dependencies](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/project/Dependencies.scala).~
- [x] Get two thumbsworth of PR review
- [x] Verify all tests go green, including CI tests
- [x] **Squash commits and merge** to develop (branches are automatically deleted after merging)
- [x] Inform other teams of any substantial changes via Slack and/or email [thread](https://broadinstitute.slack.com/archives/C03BCARP494/p1709757377503109)

[AJ-1372]: https://broadworkbench.atlassian.net/browse/AJ-1372?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by jladieu** [comment]
/jenkins retest for timeout in auth test after rebase

**Comment by jladieu** [comment]
FYI, this _did_ pass Jenkins previously, so the current failure after rebase is very likely to be a flake.

File: automation/build.sbt [hunk 1]
```diff
@@ -1,8 +1,7 @@
-import Settings._
+import Settings.*
```
Comment (davidangb) [inline automation/build.sbt:1]: did some tool suggest this change? `*` is Scala 3 syntax; `_` is Scala 2 syntax, and Rawls is still Scala 2: https://docs.scala-lang.org/scala3/reference/changed-features/imports.html

Same comment goes for `Testing.scala`

_edited to add: I guess if it works you're using the "Allow import x.{*, given} under -Xsource:3 (#9724 by @smarter)" feature from [2.13.7](https://github.com/scala/scala/releases/tag/v2.13.7)_

**Review by davidangb — APPROVED** [review]
one thought inline, no changes requested

File: automation/build.sbt (hunk 1 above)
Comment (jladieu) [reply automation/build.sbt:1]: I honestly have no idea where this came from (I'm guessing the IDE did this automatically?), but when I saw a passing Jenkins test...I didn't question it 🙈

**Comment by jladieu** [comment]
@dinvlad I tagged you on this as you inspired it about 5 months ago...finally got back around to it!

**Review by calypsomatic — APPROVED** [review]

**Pull request merged**